package.json

```diff
@@ -54,6 +54,7 @@
     "nunjucks": "^3.2.0",
     "passport": "^0.4.1",
     "passport-local": "^1.0.0",
+    "path": "^0.12.7",
```

Comment:
Everything looks good! If you want you can delete 'path' form your package.json, because the path is also a module in node